.github/workflows/memory-leak-testing.yml

```diff
@@ -30,6 +30,9 @@ jobs:
       - name: Install pnpm
         run: npm install -g pnpm
 
+      - name: Install memlab
+        run: npm install -g memlab
```

Comment:
for what you make installing of memlab? we already have it in package.json, remove it!

Reply:
This line was already in the file before I started working on it. I noticed that it was unnecessary, but when the tests didn't run in CI, I put it back to determine if it was causing the issue.